### PR TITLE
Remove unneeded variables from project build

### PR DESCRIPTION
**Pull request by snikopoulos**

### CODE REVIEW

File: src/org/usfirst/frc4048/commands/arm/GrabCube.java [hunk 1]
```diff
@@ -46,6 +46,7 @@ protected void end() {
     // Called when another command which requires one or more of the same
     // subsystems is scheduled to run
     protected void interrupted() {
+    	callback.doCancel(true);
```
Comment (armadilloz) [inline src/org/usfirst/frc4048/commands/arm/GrabCube.java:49]: Not directly related to your changes, but this command needs:
- timeout

File: src/org/usfirst/frc4048/commands/arm/MoveArm.java [hunk 2]
```diff
@@ -51,6 +51,7 @@ protected void end() {
     // Called when another command which requires one or more of the same
     // subsystems is scheduled to run
     protected void interrupted() {
+    	callback.doCancel(true);
```
Comment (armadilloz) [inline src/org/usfirst/frc4048/commands/arm/MoveArm.java:54]: Missing requires

File: src/org/usfirst/frc4048/commands/arm/GrabCube.java (hunk 1 above)
Comment (armadilloz) [reply src/org/usfirst/frc4048/commands/arm/GrabCube.java:49]: Missing requires

File: src/org/usfirst/frc4048/commands/getcube/CancelIfCubeInClaw.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+import org.usfirst.frc4048.Robot;
+import org.usfirst.frc4048.commands.GroupCommandCallback;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+public class CancelIfCubeInClaw extends Command {
+	private final GroupCommandCallback callback;
+
+	public CancelIfCubeInClaw(GroupCommandCallback callback) {
```
Comment (armadilloz) [inline src/org/usfirst/frc4048/commands/getcube/CancelIfCubeInClaw.java:11]: Missing requires

**Review by armadilloz — APPROVED** [review]

File: src/org/usfirst/frc4048/commands/arm/GrabCube.java (hunk 1 above)
Comment (snikopoulos) [reply src/org/usfirst/frc4048/commands/arm/GrabCube.java:49]: done

File: src/org/usfirst/frc4048/commands/arm/MoveArm.java (hunk 2 above)
Comment (snikopoulos) [reply src/org/usfirst/frc4048/commands/arm/MoveArm.java:54]: done

File: src/org/usfirst/frc4048/commands/getcube/CancelIfCubeInClaw.java (hunk 3 above)
Comment (snikopoulos) [reply src/org/usfirst/frc4048/commands/getcube/CancelIfCubeInClaw.java:11]: done

**Pull request merged**